README.md

```diff
@@ -1,13 +1,238 @@
+The user can then choose to start exploring the Ecosystem
+
+```java
+    AppsDiscoveryAlertDialog dialog = new AppsDiscoveryAlertDialog(context);
+    dialog.show();
+```
+
+You can create any button that follows your application style and on a click of a button, it starts the whole experience
+by starting an activity *AppsDiscoveryActivity* 
+
+```java
+someButton.setOnClickListener(v -> {
+            Intent intent = AppsDiscoveryActivity.Companion.getIntent(context)
+             startActivity(intent);
+        });
+```
+
+## Design & UX
+
+For design & UX recommendations and tips on the way to give your users the best experience please follow  [https://discover.kin.org/ux_guidlines_v1.pdf](https://discover.kin.org/ux_guidlines_v1.pdf).
+
+
+## Send Kin
+In order for your app to be able to send Kin
+
+- In your root project, add a new package named *kindiscover*
+- In kindiscover directory - create a service class named *SendKinService*, declare it in a *manifest.xml* file
+and configure the service as not to be exported
+
+```xml
+<application>
+    ...
+    <service android:name=".kindiscover.SendKinService"
+             android:exported="false" />
+</application>
+```
+
+**Notice!** it is important to define the service as **not to be exported**
+
+
+
+The *SendKinService* class must extends the abstract service *SendKinServiceBase*
+and implement the two abstract methods:
+- *transferKin* 
+- *getCurrentBalance* 
+
+
+```java
+public class SendKinService extends SendKinServiceBase {
+    @NonNull
+    @Override
+    public KinTransferComplete transferKin(@NonNull String toAddress, int amount, @NonNull String memo) throws KinTransferException {
+       
+        //perform the transfer with the given parameters
+       
+        //if transfer fails throw KinTransferException
+          throw new KinTransferException("sender address", "transfer error details");
+          //if transfer completes
+        return new KinTransferComplete("sender address", "transaction id", "transaction memo");
+    }
+
+    @Override
+    public BigDecimal getCurrentBalance() throws BalanceException {
+        
+        //get balance from the wallet
+       
+       double balance = ...getBalance();
+       
+       //if fails to get balance throw BalanceException
+         throw new BalanceException("error details");
+         //else return the balance
+       return new BigDecimal(balance);
+    }
+}
+```
+
+These methods are asynchronous and can perform long operations
+
+
+
+## Receive Kin
+In order for an app to be able to receive Kin from other apps
+
+- In your root project add new package named *kindiscover*
+- In the kindiscover directory - create an activity class *AccountInfoActivity* and declare it in *manifest.xml* file
+
+
+```xml
+<application>
+    ...
+    <activity android:name=".kindiscover.AccountInfoActivity"
+              android:exported="true"/>       
+</application>
+```
+Very important to define the activity to be exported
```

Comment:
The activity *must* be exported so that it can be reached from the app that wishes to send Kin to your app.